src/Orckestra.Composer.Search/Providers/FromPriceProvider.cs

```diff
@@ -168,7 +173,13 @@ private static string GetPriceListId(ProductDocument document)
 
         private string GetDisplayPrice(double? price)
         {
-            return !price.HasValue ? null : price.Value.ToString("C2", ComposerContext.CultureInfo);
+            if (price.HasValue)
+            {
+                decimal.TryParse(price.Value.ToString(CultureInfo.InvariantCulture), out var priceVal);
+                return LocalizationProvider.FormatPrice(priceVal, ComposerContext.CurrencyIso);
```

Comment:
Why not just

`LocalizationProvider.FormatPrice(price.Value, ComposerContext.CurrencyIso);`

?

Reply:
price.Value double - but in FormatPrice method we take decimal